src/pages/PassportRedemption/Faq.tsx

```diff
@@ -10,13 +10,12 @@ type Props = {
 };
 
 const Faq = ({ showFaq, toggleView }: Props) => {
-  const history = useHistory();
   const { id } = useParams();
 
-  const goToPassPort = (e) => {
-    history.push(`/passport/${id}/tickets`);
+  const goToFoodCrawlInfo = (e) => {
+    window.location.href = "https://www.sendchinatownlove.com/food-crawl.html";
```

Comment:
I think we wanted external links to be opened in a new window?

Reply:
ooh good call, fixed